cmake/ctest/drivers/atdm/cts1/drivers/Trilinos-atdm-cts1-intel-debug-openmp.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#export SALLOC_CTEST_TIME_LIMIT_MINUTES=1:00:00
+if [ "${Trilinos_TRACK}" == "" ] ; then
+  export Trilinos_TRACK=ATDM
```

Comment:
To be safe, should start out with "Experimental" and then we can upgrade as we see things are clean.